src/bundle/Resources/config/dashboard.yml

```diff
@@ -0,0 +1,87 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    EzSystems\HybridPlatformUi\Dashboard\:
+        resource: '../../../lib/Dashboard/*'
+
+    EzSystems\HybridPlatformUi\Dashboard\Dashboard:
```

Comment:
Yes, it is. :) Our Dashboard class definition does not require anything else, et still have to be defined to be injectable by Dependency Injection layer.